src/averageAges.js

```diff
@@ -15,11 +15,17 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const filteredArrayOfMen = people.filter((person) =>
+    century
+      ? person.sex === 'm' && Math.ceil(person.died / 100) === century
+      : person.sex === 'm'
+  );
+
+  const totalAge = filteredArrayOfMen.reduce((total, person) => {
+    return total + (person.died - person.born);
+  }, 0);
```

Comment:
Create separate helper function to get Avarage age - so you can use it here and in `calculateWomenAverageAge` withou code duplicating